src/quicklist.c

```diff
@@ -1739,54 +1741,52 @@ int quicklistTest(int argc, char *argv[], int accurate) {
         }
 
         TEST_DESC("add to tail 5x at compress %d", options[_i]) {
-        for (int f = optimize_start; f < 32; f++) {
-                quicklist *ql = quicklistNew(f, options[_i]);
+        for (int f = 0; f < fill_count; f++) {
+                quicklist *ql = quicklistNew(fills[f], options[_i]);
```

Comment:
let me check if i understand, these loops were going from -5 to 32 (total of 37 iterations), and now they run 18 iterations, using values from -5 to 1024?
is `66` a typo?
not sure we need that many iterations on the high end.
i guess we can tone it down further more to some 10 or 12 iterations?
i guess there is some value of some values that are non-power of two?

Reply:
1. Yes, now only up to 18 iterations.
2. `66` is to cover the original test
```
if (fills[f] == 1)
    ql_verify(ql, 66, 66, 1, 1);
else if (fills[f] == 32)
    ql_verify(ql, 3, 66, 32, 2);
else if (fills[f] == 66)
    ql_verify(ql, 1, 66, 66, 66);
```
3. Since the old fill maximum is 1024, I'll remove and add some special fills.

Reply:
ohh, i missed the fact that some iterations were running till 72 and 1024 (thought they all run from -5 to 32)